ci-operator/step-registry/rosa/conf/idp/htpasswd/rosa-conf-idp-htpasswd-commands.sh

```diff
@@ -105,10 +105,20 @@ done
 
 echo "Waiting for cluster-admin ready..."
 start_time=$(date +"%s")
+
+# check if there is already cluster admin IDP when creating cluster
+cluster_idp=$(rosa list idp -c ${CLUSTER_ID} -o json | jq -r ".[].name")
```

Comment:
Don't need to list the idp anymore. As the whole script is acting on the new created IDP. Just hardcode the new created IDP name to a variable and then list idp with name filter or describe idp.